### PR TITLE
Common tree highlight groups

### LINKED ISSUE #157: NeoTree showing poor contrast
Hi, I started using Nordic and I think it's a fantastic colorscheme, but I've had a problem with some NeoTree colors.
Screenshot from 2024-09-23 11-42-31
As you can see in this screenshot, text color of a git renamed file has some very low contrast (and the folders in which changes have been made). Also when I add a new file or dir through NeoTree, text also has very low contrast as shown in this image:
image

### PR DESCRIPTION
**Pull request by bini-x**
Closes #157.

### CODE REVIEW

**Comment by AlexvZyl** [comment]
Can you please rather make the changes from the dev branch?

**Comment by AlexvZyl** [comment]
@5-pebbles I forgot this was going to happen with our dev setup.  I assume it's fine having people submit PRs from the dev branch.

**Comment by AlexvZyl** [comment]
@bini-x Wait, leave this PR as is for now.

**Comment by bini-x** [comment]
Ok

**Comment by AlexvZyl** [comment]
Run this in the root of nordic:

`stylua --config-path=.stylua.toml .`

You might have to install stylua first.

**Comment by bini-x** [comment]
I just saw what my formatter did. sorry. i fixed it all now

**Comment by bini-x** [comment]
@AlexvZyl

**Comment by AlexvZyl** [comment]
Great, now can you share a preview of the new changes?

**Comment by bini-x** [comment]
yes
image
you can see the color of root name is now dark blue (blue0 of color palette), git untracked is now red_b and in the next image you can see the dialog that opens when you add/rename/delete a file/directory is orange_b:
image

**Comment by bini-x** [comment]
@AlexvZyl any comments?

**Comment by AlexvZyl** [comment]
Will reply, gonna go mountain biking

**Comment by bini-x** [comment]
image
just changed some colors i thought were bad earlier

**Comment by AlexvZyl** [comment]
Definitely like iteration two more :)

**Comment by AlexvZyl** [comment]
Check the tests.  Somewhere you misspelled `fg`.

@5-pebbles These tests are lovely

**Comment by AlexvZyl** [comment]
Ah no, you used `orange` directly.  Orange is a table.

**Comment by bini-x** [comment]
Oh okay. What's the name of the first orange in palette?

**Comment by bini-x** [comment]
I'll try to find and I'll fix it

**Comment by AlexvZyl** [comment]
https://github.com/AlexvZyl/nordic.nvim/blob/main/lua/nordic/colors/nordic.lua

**Comment by AlexvZyl** [comment]
Consider adding https://github.com/folke/lazydev.nvim to your setup.

**Comment by bini-x** [comment]
@AlexvZyl fixed

**Comment by bini-x** [comment]
btw alex what's the name of the font used here:
image

**Comment by AlexvZyl** [comment]
> btw alex what's the name of the font used here: ...

That is JetBrains Mono https://www.jetbrains.com/lp/mono/

**Comment by AlexvZyl** [comment]
So, I have been thinking about this and I have a suggestion (you can do it if you want).

Since there are multiple tree plugins, I think we should define some common colors.  For example:

```lua
TreeFiles = ...
TreeGitIcons = ...
TreeHeader = ...
```

And then have NeoTree and NvimTree link to these highlights.

**Comment by bini-x** [comment]
Yes I can do it. I'll just have to check the Nvim-Tree styles first

**Comment by bini-x** [comment]
I haven't used nvimtree before, would you think that the colors I added to neotree would be good to add to nvimtree, so I wouldn't have to change anything just add these lines that you mentioned and link them

**Comment by AlexvZyl** [comment]
> So, I have been thinking about this and I have a suggestion (you can do it if you want).
> 
> Since there are multiple tree plugins, I think we should define some common colors. For example:
> 
> ```lua
> TreeFiles = { ... },
> TreeFolders = { ... },
> TreeGitIcons = { ... },
> TreeHeader = { ... },
> ```
> 
> And then have NeoTree and NvimTree link to these highlights. Otherwise we have to manage multiple groups to have similar highlights across plugins.
> 
> You can see I already tried to do this in NeoTree by linking to NvimTree.

@5-pebbles Note this, @bini-x wants to try it.

**Comment by bini-x** [comment]
@AlexvZyl done

**Comment by AlexvZyl** [comment]
Nice.  Can you try and link all of the groups?

**Comment by bini-x** [comment]
yeah I can do that, but I didn't see more groups that are the same. I'll try to find them and let you know. If I can't please tell me them if you can find em.

**Comment by bini-x** [comment]
for example NeoTree has FileIcon but NvimTree doesn't:
image

**Comment by AlexvZyl** [comment]
I did not add all of the hl groups to that file.

Try: `:Telescope highlights`

**Comment by AlexvZyl** [comment]
Also quick tip, in that screenshot you could do:
```
/FileIcon
```

**Comment by bini-x** [comment]
@AlexvZyl you can check it now, because I wasn't sure if these were groups or not

**Comment by AlexvZyl** [comment]
That looks good.  Will test it locally later.

**Comment by AlexvZyl** [comment]
Closes #157

File: lua/nordic/groups/integrations.lua [hunk 1]
```diff
@@ -97,39 +97,83 @@ function M.get_groups()
     G.MiniStatuslineFileinfo = { bg = LC.normal.b.bg, fg = LC.normal.b.fg }
     G.MiniStatuslineInactive = { bg = LC.inactive.a.bg, fg = LC.inactive.a.fg, bold = LC.inactive.a.gui == 'bold' }
 
+    -- Tree Groups
+    G.TreeNormal = { fg = C.fg, bg = C.bg }
+    G.TreeNormalNC = { fg = C.fg, bg = C.bg }
+    G.TreeRootFolder = { fg = C.gray4 }
+    G.TreeRootName = { fg = C.white2 }
```
Comment (5-pebbles) [inline lua/nordic/groups/integrations.lua:104]: This is too close to TreeNormal for me to tell the difference. It just looks a little off. However, that might just be a me problem.

**Comment by AlexvZyl** [comment]
I actually don't use those tree plugins anymore.  Would you mind sharing screenshots?

**Comment by bini-x** [comment]
yes, I can but just for NeoTree because I don't use nvimTree

**Comment by bini-x** [comment]
image
you can see the dialog prompt has the bg white now, git untracked is orange and rootname is white.

File: lua/nordic/groups/integrations.lua [hunk 2]
```diff
@@ -97,39 +97,83 @@ function M.get_groups()
...
+    G.TreeGitConflict = { fg = C.magenta.bright }
+    G.TreeGitModified = { fg = C.git.change }
+    G.TreeGitDirty = { fg = C.gray4 }
+    G.TreeGitAdded = { fg = C.git.add }
+    G.TreeGitNew = { fg = C.gray4 }
+    G.TreeGitDeleted = { fg = C.gray4 }
+    G.TreeGitStaged = { fg = C.gray4 }
+    G.TreeGitUntracked = { fg = C.orange.base }
+    G.TreeTitleBar = { fg = C.gray2, bg = C.white3 }
+    G.TreeFloatBorder = { fg = C.white3, bg = C.gray0 }
```
Comment (5-pebbles) [inline lua/nordic/groups/integrations.lua:117]: These seem out of the norm for this theme. I can't really think of what color should be used, but I will mull it over.

File: lua/nordic/groups/integrations.lua [hunk 3]
```diff
@@ -97,39 +97,83 @@ function M.get_groups()
...
+    G.TreeCursorLine = { bg = C.gray1 }
```
Comment (5-pebbles) [inline lua/nordic/groups/integrations.lua:118]: what about using `bg_selected` here its already what is used for `G.PmenuSel = { bg = C.bg_selected } -- Popup menu: selected item.`

**Review by 5-pebbles — COMMENTED** [review]
I like the merger of tree plugins a lot!

File: lua/nordic/groups/integrations.lua (hunk 3 above)
Comment (bini-x) [reply lua/nordic/groups/integrations.lua:118]: all I changed was the ones I mentioned in the last comment. I didn't actually check how the other colors looked.

Comment (bini-x) [reply lua/nordic/groups/integrations.lua:118]: I just "grouped" them as alex told me his idea

Comment (5-pebbles) [reply lua/nordic/groups/integrations.lua:118]: I meant to only select the `TreeCursorLine`...

Comment (bini-x) [reply lua/nordic/groups/integrations.lua:118]: oh okay. so as far as I can understand you want to only change the color of TreeCursorLine right?

Comment (5-pebbles) [reply lua/nordic/groups/integrations.lua:118]: > I just "grouped" them as alex told me his idea

I did not see your message. If you would like we can change that it a different PR.



> oh okay. so as far as I can understand you want to only change the color of TreeCursorLine right?

Correct.

Comment (bini-x) [reply lua/nordic/groups/integrations.lua:118]: I can change that color. You want to change it to C.bg_selected right?

Comment (bini-x) [reply lua/nordic/groups/integrations.lua:118]: i don't actually know if I get what you mean here

> I did not see your message. If you would like we can change that it a different PR.

Comment (bini-x) [reply lua/nordic/groups/integrations.lua:118]: @5-pebbles should I change the TreeCursorLine bg to C.bg_selected and push it?

Comment (5-pebbles) [reply lua/nordic/groups/integrations.lua:118]: Alright: GitHub is not showing your messages unless I refresh the page which is weird. So I did not notice that you had commented on my message.

I think this line: `G.TreeCursorLine = { bg = C.gray1 }` should be changed to this `G.TreeCursorLine = { bg = C.bg_selected }`.

However it was like this before your pull request was opened and so I was offering to open a different PR to fix it. After yours has been merged.

Comment (5-pebbles) [reply lua/nordic/groups/integrations.lua:118]: > @5-pebbles should I change the TreeCursorLine bg to C.bg_selected and push it?

yes

Comment (bini-x) [reply lua/nordic/groups/integrations.lua]: done

Comment (bini-x) [reply lua/nordic/groups/integrations.lua]: > However it was like this before your pull request was opened and so I was offering to open a different PR to fix it. After yours has been merged.
 That would be no problem for me but since I pushed it now, this is no more needed right?

Comment (5-pebbles) [reply lua/nordic/groups/integrations.lua]: great thank you! ... but your formatter did the thing again... lol

Comment (bini-x) [reply lua/nordic/groups/integrations.lua]: oh shit forgot to turn it off hahahahaha
i'll fix it now

Comment (5-pebbles) [reply lua/nordic/groups/integrations.lua]: > > However it was like this before your pull request was opened and so I was offering to open a different PR to fix it. After yours has been merged.
> 
> That would be no problem for me but since I pushed it now, this is no more needed right?

Correct

Comment (bini-x) [reply lua/nordic/groups/integrations.lua]: fixed it

File: lua/nordic/groups/integrations.lua (hunk 2 above)
Comment (bini-x) [reply lua/nordic/groups/integrations.lua:117]: ok, if you got something let me know

File: lua/nordic/groups/integrations.lua (hunk 1 above)
Comment (bini-x) [reply lua/nordic/groups/integrations.lua:104]: idk I was just based on how the "nord" theme managed their colors

Comment (bini-x) [reply lua/nordic/groups/integrations.lua:104]: also if you have something in mind for this part tell me and i'll change it

File: lua/nordic/groups/integrations.lua (hunk 2 above)
Comment (AlexvZyl) [reply lua/nordic/groups/integrations.lua:117]: They should probably use `bg_float` (will check now what it is called) since float colors depends on the config

Comment (AlexvZyl) [reply lua/nordic/groups/integrations.lua:117]: Take a look here https://github.com/AlexvZyl/nordic.nvim/blob/main/lua/nordic/colors/init.lua

Comment (bini-x) [reply lua/nordic/groups/integrations.lua:117]: it's bg_float, fg_float for TreeTitleBar and fg_float_border, bg_float_border for TreeFloatBorder as I can see

Comment (5-pebbles) [reply lua/nordic/groups/integrations.lua:117]: > Take a look here https://github.com/AlexvZyl/nordic.nvim/blob/main/lua/nordic/colors/init.lua

I am not sure about that they are more like a title: 
image

Comment (5-pebbles) [reply lua/nordic/groups/integrations.lua:117]: The most obvious titles we have are for telescope but I am not sure those work here either:
image

Comment (bini-x) [reply lua/nordic/groups/integrations.lua:117]: i'd say the blue one looks good. I can try them and send screenshots and tell me which one is better

Comment (bini-x) [reply lua/nordic/groups/integrations.lua:117]: can you just give me the colors for fg and bg?

Comment (5-pebbles) [reply lua/nordic/groups/integrations.lua:117]: Not the right text color but just background examples:
image
image

Comment (5-pebbles) [reply lua/nordic/groups/integrations.lua:117]: > can you just give me the colors for fg and bg?

This line: https://github.com/AlexvZyl/nordic.nvim/blob/1ee4044077059b34eacd18d93613495a602c131f/lua/nordic/groups/integrations.lua#L259

orange.base and blue2

Comment (bini-x) [reply lua/nordic/groups/integrations.lua:117]: ok, I would say that the blue one fits better. if y'all think otherwise tell me

Comment (5-pebbles) [reply lua/nordic/groups/integrations.lua:117]: All the other (maybe) non reserved non grayscale ones:
image
image
image
image

Comment (bini-x) [reply lua/nordic/groups/integrations.lua:117]: idk about these, the orange one and the blue were better

Comment (5-pebbles) [reply lua/nordic/groups/integrations.lua:117]: > ok, I would say that the blue one fits better. if y'all think otherwise tell me

I agree with you and if a few people don't its fairly trivial to change it in their config. Assuming that's alright with Alex.

Comment (5-pebbles) [reply lua/nordic/groups/integrations.lua:117]: Make sure to use the `bold = true` and `fg = C.black0` options.

Comment (bini-x) [reply lua/nordic/groups/integrations.lua:117]: image
this is how it looks in my config

Comment (bini-x) [reply lua/nordic/groups/integrations.lua:117]: i think it looks great

Comment (5-pebbles) [reply lua/nordic/groups/integrations.lua:117]: That looks great to me! : )

Comment (bini-x) [reply lua/nordic/groups/integrations.lua:117]: @AlexvZyl what do you think?

Comment (bini-x) [reply lua/nordic/groups/integrations.lua:117]: image
@5-pebbles this is it right?

Comment (5-pebbles) [reply lua/nordic/groups/integrations.lua:117]: > image @5-pebbles this is it right?

I don't think the border should be bold.

Comment (bini-x) [reply lua/nordic/groups/integrations.lua:117]: yeah that's true

Comment (bini-x) [reply lua/nordic/groups/integrations.lua:117]: and what do y'all think about directories being blue?
image
just tried it and I think it doesn't look bad

Comment (5-pebbles) [reply lua/nordic/groups/integrations.lua:117]: > and what do y'all think about directories being blue? image just tried it and I think it doesn't look bad

I don't think that's a good idea. Mostly just because it defies peoples expectations and is easy for them to set in their individual configs.

Comment (bini-x) [reply lua/nordic/groups/integrations.lua:117]: yeah that's true, I thought about it and honestly makes it a little confusing for someone. and yeah, if someone wants it, he can use it for himself

Comment (bini-x) [reply lua/nordic/groups/integrations.lua:117]: im just waiting for alex to confirm the new colors for titlebar and floatborder so i can push it

Comment (5-pebbles) [reply lua/nordic/groups/integrations.lua:117]: Its like 9PM for him so that might be a while

Comment (bini-x) [reply lua/nordic/groups/integrations.lua:117]: Me too lol

File: lua/nordic/groups/integrations.lua (hunk 1 above)
Comment (5-pebbles) [reply lua/nordic/groups/integrations.lua:104]: I am thinking it doesn't need it's own color. It's not like it's hard to identify (always the top one).

Comment (bini-x) [reply lua/nordic/groups/integrations.lua:104]: you talking about the TreeNormal or TreeRootName?

Comment (5-pebbles) [reply lua/nordic/groups/integrations.lua:104]: > you talking about the TreeNormal or TreeRootName?

TreeRootName

Comment (5-pebbles) [reply lua/nordic/groups/integrations.lua:104]: I think I like that best. What about just making it `G.TreeRootName = G.TreeNormal`

Comment (bini-x) [reply lua/nordic/groups/integrations.lua:104]: done

Comment (bini-x) [reply lua/nordic/groups/integrations.lua:104]: @5-pebbles is there a way to test out these changes I made before merging them to main. I mean I tried this in my config but how do I try the theme as a "whole" with the new changes?

Comment (5-pebbles) [reply lua/nordic/groups/integrations.lua:104]: > @5-pebbles is there a way to test out these changes I made before merging them to main. I mean I tried this in my config but how do I try the theme as a "whole" with the new changes?

We have automated tests which you should be able to run locally with `nvim --headless -c "lua require('nordic.tests').run_tests()" -c 'q' 2>&1` those are also automatically ran in a VM when you push changes to PR.

Comment (5-pebbles) [reply lua/nordic/groups/integrations.lua:104]: That doesn't necessarily mean everything is working it just means you haven't broken anything in a way I have already thought of...

Comment (bini-x) [reply lua/nordic/groups/integrations.lua]: i pushed so whenever y'all can run the tests and see if it works

Comment (5-pebbles) [reply lua/nordic/groups/integrations.lua]: The tests didn't get it but there is a bug and it's my fault...
image

I was wrong when I said the `= G.TreeNormal` thing use this instead: `    G.TreeRootName = { fg = C.fg }
`

Comment (5-pebbles) [reply lua/nordic/groups/integrations.lua]: Apparently that overrides the selected background...

Comment (bini-x) [reply lua/nordic/groups/integrations.lua]: done

**Review by 5-pebbles — DISMISSED** [review]
Everything looks good to me : )

**Comment by bini-x** [comment]
@AlexvZyl have you seen the final result yet?

**Comment by AlexvZyl** [comment]
> @AlexvZyl have you seen the final result yet?

I don't miss a thing :) I promise I will properly check when I get the chance.

**Comment by bini-x** [comment]
@5-pebbles what do you think about making the root directory's name bold? 
image

**Comment by bini-x** [comment]
because in cases like this, if it isn't bold it looks the same as file names:
image

**Comment by 5-pebbles** [comment]
> @5-pebbles what do you think about making the root directory's name bold? image

I actually really like that. Go for it!

**Review by 5-pebbles — APPROVED** [review]
That looks great!

**Comment by geofflane** [comment]
👍🏼 To this. Just noticed the same issue.

File: lua/nordic/groups/integrations.lua [hunk 4]
```diff
@@ -97,39 +97,83 @@ function M.get_groups()
     G.MiniStatuslineFileinfo = { bg = LC.normal.b.bg, fg = LC.normal.b.fg }
     G.MiniStatuslineInactive = { bg = LC.inactive.a.bg, fg = LC.inactive.a.fg, bold = LC.inactive.a.gui == 'bold' }
 
+    -- Tree Groups
+    G.TreeNormal = { fg = C.fg, bg = C.bg }
+    G.TreeNormalNC = { fg = C.fg, bg = C.bg }
+    G.TreeRootFolder = { fg = C.gray4 }
+    G.TreeRootName = { fg = C.fg, bold = true }
```
Comment (AlexvZyl) [inline lua/nordic/groups/integrations.lua:104]: I think these should be the same thing?

Comment (AlexvZyl) [inline lua/nordic/groups/integrations.lua:104]: I think these two are the same thing?

File: lua/nordic/groups/integrations.lua [hunk 5]
```diff
@@ -97,39 +97,83 @@ function M.get_groups()
...
+    G.TreeFileIcon = { fg = C.blue2 }
+    G.TreeFileNameOpened = { fg = C.fg }
+    G.TreeSpecialFile = { fg = C.magenta.bright }
+    G.TreeGitConflict = { fg = C.magenta.bright }
```
Comment (AlexvZyl) [inline lua/nordic/groups/integrations.lua:108]: Conflict rather red?

File: lua/nordic/groups/integrations.lua [hunk 6]
```diff
@@ -97,39 +97,83 @@ function M.get_groups()
...
+    G.TreeGitModified = { fg = C.git.change }
+    G.TreeGitDirty = { fg = C.gray4 }
+    G.TreeGitAdded = { fg = C.git.add }
+    G.TreeGitNew = { fg = C.gray4 }
+    G.TreeGitDeleted = { fg = C.gray4 }
+    G.TreeGitStaged = { fg = C.gray4 }
+    G.TreeGitUntracked = { fg = C.orange.base }
```
Comment (AlexvZyl) [inline lua/nordic/groups/integrations.lua:115]: IMO these "identifiers" should either:
1. All be subtle and grey (I prefer this)
2. Or have their respective colors.

@5-pebbles Opinion?

File: lua/nordic/groups/integrations.lua [hunk 7]
```diff
@@ -97,39 +97,83 @@ function M.get_groups()
...
+    G.TreeTitleBar = { fg = C.black0, bg = C.blue2, bold = true }
```
Comment (AlexvZyl) [inline lua/nordic/groups/integrations.lua:116]: I think this should maybe link to Winbar?

**Review by AlexvZyl — CHANGES_REQUESTED** [review]

File: lua/nordic/groups/integrations.lua [hunk 8]
```diff
@@ -97,39 +97,83 @@ function M.get_groups()
...
+    G.TreeFloatBorder = { fg = C.blue2, bg = C.gray0}
+    G.TreeCursorLine = { bg = C.bg_selected }
+    G.TreeCursor = { bg = C.none, fg = C.none }
+    G.TreeFolderIcon = { fg = C.yellow.dim }
+    G.TreeIndentMarker = { fg = C.gray4 }
+    G.TreeSymlink = { fg = C.blue2 }
+    G.TreeFolderName = { fg = C.blue1 }
+    G.TreeWinSeparator = { fg = C.bg_dark, bg = C.bg }
```
[the remark this answers is not in the document]
Comment (bini-x) [reply lua/nordic/groups/integrations.lua:124]: Thanks for your reply, Alex!
Couldn't reply earlier because I had some other things to do.
I'll see what I can do and I'll try to finish it as soon as I can.

File: lua/nordic/groups/integrations.lua (hunk 6 above)
Comment (5-pebbles) [reply lua/nordic/groups/integrations.lua:115]: I would personally prefer colors as the default.

My repos are usually an unstaged mess, and the colors help prevent the spread of chaos, but that's more of a "me problem".

I think either way works; the people who don't like whatever we pick will just change it in their own configs. Though, it would be worth adding the unchosen variation to the list of common configurations that I plan on writing someday.

**TLDR:** I don't know which is better...

File: lua/nordic/groups/integrations.lua (hunk 7 above)
Comment (bini-x) [reply lua/nordic/groups/integrations.lua:116]: the whole 4 lines link to Winbar right?

Comment (bini-x) [reply lua/nordic/groups/integrations.lua:116]: or just the titlebar

File: lua/nordic/groups/integrations.lua [hunk 9]
```diff
@@ -97,39 +97,83 @@ function M.get_groups()
...
+    G.TreeGitConflict = { fg = C.magenta.bright }
+    G.TreeGitModified = { fg = C.git.change }
+    G.TreeGitDirty = { fg = C.gray4 }
+    G.TreeGitAdded = { fg = C.git.add }
+    G.TreeGitNew = { fg = C.gray4 }
+    G.TreeGitDeleted = { fg = C.gray4 }
+    G.TreeGitStaged = { fg = C.gray4 }
+    G.TreeGitUntracked = { fg = C.orange.base }
+    G.TreeTitleBar = { fg = C.black0, bg = C.blue2, bold = true }
+    G.TreeFloatBorder = { fg = C.blue2, bg = C.gray0}
```
[the remark this answers is not in the document]
Comment (bini-x) [reply lua/nordic/groups/integrations.lua:117]: can you tell me the float colors?

File: lua/nordic/groups/integrations.lua (hunk 8 above)
[the remark this answers is not in the document]
Comment (bini-x) [reply lua/nordic/groups/integrations.lua:124]: i can't find the existing window separator

File: lua/nordic/groups/integrations.lua (hunk 6 above)
Comment (bini-x) [reply lua/nordic/groups/integrations.lua:115]: which gray do you think should they be?

File: lua/nordic/groups/integrations.lua (hunk 9 above)
[the remark this answers is not in the document]
Comment (5-pebbles) [reply lua/nordic/groups/integrations.lua:117]: You can find the extended palette values and their definitions here:
https://github.com/AlexvZyl/nordic.nvim/blob/1ee4044077059b34eacd18d93613495a602c131f/lua/nordic/colors/init.lua#L59-L63

File: lua/nordic/groups/integrations.lua (hunk 8 above)
[the remark this answers is not in the document]
Comment (5-pebbles) [reply lua/nordic/groups/integrations.lua:124]: just use: `link = "WinSeparator"`
see definition: https://github.com/AlexvZyl/nordic.nvim/blob/1ee4044077059b34eacd18d93613495a602c131f/lua/nordic/groups/native.lua#L135

File: lua/nordic/groups/integrations.lua (hunk 7 above)
Comment (5-pebbles) [reply lua/nordic/groups/integrations.lua:116]: > is it just link = "Winbar", because I can't find anything with the name "Winbar" in this file

Yup, that should work:
https://github.com/AlexvZyl/nordic.nvim/blob/1ee4044077059b34eacd18d93613495a602c131f/lua/nordic/groups/native.lua#L179

For future reference if you click the search bar on GitHub via this pr it will search the current repos contents. You can do the same locally with something like ripgrep or telescopes built in ripgrep picker.

File: lua/nordic/groups/integrations.lua (hunk 8 above)
[the remark this answers is not in the document]
Comment (AlexvZyl) [reply lua/nordic/groups/integrations.lua:124]: Sorry, should just have told you what the hl is :joy:

File: lua/nordic/groups/integrations.lua (hunk 6 above)
Comment (AlexvZyl) [reply lua/nordic/groups/integrations.lua:115]: I think most people expect colors.  So go with that.

File: lua/nordic/groups/integrations.lua (hunk 8 above)
[the remark this answers is not in the document]
Comment (bini-x) [reply lua/nordic/groups/integrations.lua:124]: Np, I'm thinking now how could I not find the window separator, it was there the whole time...😅

**Comment by bini-x** [comment]
@AlexvZyl added the requested changes, hopefully I didn't forget anything

**Comment by bini-x** [comment]
i'm not so sure about line 116 tho

**Comment by AlexvZyl** [comment]
Sorry for disappearing, again... been a rough year.

Will check the changes again when I am back at my PC (around 2 Jan).

File: lua/nordic/groups/integrations.lua [hunk 10]
```diff
@@ -97,39 +97,82 @@ function M.get_groups()
...
+    G.TreeGitConflict = { fg = C.red.base }
+    G.TreeGitModified = { fg = C.git.change }
+    G.TreeGitDirty = { fg = C.gray4 }
+    G.TreeGitAdded = { fg = C.git.add }
+    G.TreeGitNew = { fg = C.gray4 }
+    G.TreeGitDeleted = { fg = C.gray4 }
+    G.TreeGitStaged = { fg = C.gray4 }
+    G.TreeGitUntracked = { fg = C.orange.base }
+    G.TreeTitleBar = { link = "WinBar"}
+    G.TreeFloatBorder = { fg = C.border_fg, bg = C.bg_float }
```
Comment (5-pebbles) [inline lua/nordic/groups/integrations.lua:116]: I think we mislead you on this one:
```suggestion
    G.TreeFloatBorder = { link = "FloatBorder" }
```

File: lua/nordic/groups/integrations.lua (hunk 6 above)
Comment (5-pebbles) [reply lua/nordic/groups/integrations.lua:114]: I think these could use some better colors, but they were like that before this PR... If we decide to change them, that should be its own thing.

**Review by 5-pebbles — COMMENTED** [review]
I am really sorry we had you waiting on us for so long, Alex said I could merge this without him. There is one highlight which should be linked to another, I added the suggestion to this review, and you will be able to change that with the button here on GitHub (or just push like normal). 

If it all looks good, please send me one last screen and I will get this merged. Thank you very much for being patient.

**Comment by bini-x** [comment]
@5-pebbles i just committed the change you suggested. I currently don't have neovim as I had to switch to windows for some time so I can't send you a screenshot, sorry. Hopefully you can check if everything is good.

**Comment by 5-pebbles** [comment]
Alright, I will install the plugin and check it out sometime before this weekend. Thanks for working on this.

**Review by 5-pebbles — APPROVED** [review]
Alright, this weekend has passed... but I did just check it and every thing looks good!

**Comment by 5-pebbles** [comment]
*needs
